pkg/minikube/config/types.go

```diff
@@ -118,6 +118,10 @@ type KubernetesConfig struct {
 	NodeIP   string
 	NodePort int
 	NodeName string
+
+	// AutoPauseTime is used to specify how often minikube needs to check to autopause
+	// cluster.
+	AutoPauseTime time.Duration
```

Comment:
could you put this directly in Config instead of KubernetesConfig? AutoPauseTime isn't really Kubernetes configuration.